python-lib/wordcloud_visualizer.py

```diff
@@ -58,11 +56,41 @@ class WordcloudVisualizer:
     DEFAULT_BBOX_INCHES = "tight"
     DEFAULT_BACKGROUND_COLOR = "white"
 
+    DEFAULT_FONT = "NotoSansMerged-Regular-1000upem.ttf"
+    """Multilingual font created from the fusion of the following Noto Sans fonts:
+        - NotoSansDisplay-Regular
+        - NotoSansArabic-Regular
+        - NotoSansArmenian-Regular
+        - NotoSansBengali-Regular
+        - NotoSansDevanagari-Regular
+        - NotoSansHebrew-Regular
+        - NotoSansSinhala-Regular
+        - NotoSansTamil-Regular
+        - NotoSansThai-Regular
+    """
+    FONT_EXCEPTIONS_DICT = {
+        "gu": "NotoSansMerged-Regular-2048upem.ttf",
+        "kn": "NotoSansMerged-Regular-2048upem.ttf",
+        "ml": "NotoSansMerged-Regular-2048upem.ttf",
+        "te": "NotoSansMerged-Regular-2048upem.ttf",
+        "zh": "NotoSansCJKsc-Regular.otf",
```

Comment:
We had the requirements from Applied Materials to support traditional and simplified Chinese in different ways. Could we add zh-Hant for traditional Chinese?

Reply:
Logging as feature request in the backlog to avoid scope creep: https://app.clubhouse.io/dataiku/story/62001/add-traditional-chinese